src/Raven.Client/Documents/Indexes/Vector/AutoVectorOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Raven.Client.Documents.Indexes.Vector;
+
+public sealed class AutoVectorOptions : VectorOptions
+{
+    /// <summary>
+    /// Data source of embeddings
+    /// </summary>
+    public string SourceFieldName { get; set; }
+
+    public AutoVectorOptions()
+    {
+        
+    }
+    public AutoVectorOptions(AutoVectorOptions options)
+    {
+        Dimensions = options.Dimensions;
+        SourceEmbeddingType = options.SourceEmbeddingType;
+        DestinationEmbeddingType = options.DestinationEmbeddingType;
+        SourceFieldName = options.SourceFieldName;
+    }
```

Comment:
We have global and constant values for `NumberOfEdges`/`NumberOfCandidates`. Do we want to introduce configuration options for them (not granularly, just for auto indexes in the database)?

cc @ayende